custom_components/sat/serial/__init__.py

```
@@ -25,12 +27,12 @@
 class SatSerialCoordinator(SatDataUpdateCoordinator):
     """Class to manage to fetch data from the OTGW Gateway using pyotgw."""
 
-    def __init__(self, hass: HomeAssistant, port: str, config_data: Mapping[str, Any], options: Mapping[str, Any] | None = None) -> None:
+    def __init__(self, hass: HomeAssistant, config: SatConfig) -> None:
         """Initialize."""
-        super().__init__(hass, config_data, options)
+        super().__init__(hass, config)
         self.async_set_updated_data(DEFAULT_STATUS)
```

Pylint (4.0.4)

[error] Undefined variable 'DEFAULT_STATUS'

(E0602)